CVX/load_stage.sql

```diff
@@ -47,10 +47,22 @@ INSERT INTO concept_stage (
 	valid_end_date,
 	invalid_reason
 	)
-SELECT SUBSTR(full_vaccine_name, 1, 255) AS concept_name,
+SELECT CASE 
+		WHEN d.vaccinestatus = 'Non-US'
+			AND d.full_vaccine_name NOT ILIKE '%Non-US%'
```

Comment:
@MariaRohozhkina Why is is NOT like?

Reply:
just a refactoring of two lines in https://github.com/OHDSI/Vocabulary-v5.0/pull/776/commits/8af2cfb54dd0ac680dfab3e2a0ac6e49d242f2e1#diff-48bf6ee6d03fe9c21eb548122ff5d43e13ad4a933e9181ae52fd56b9c1f99441R52